standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/utils/MetaStoreUtils.java

```diff
@@ -229,6 +231,15 @@ public static boolean isExternalTable(Table table) {
     return isExternal(params);
   }
 
+  public static boolean isDbBeingFailedOver(Database db) {
+    assert (db != null);
+    Map<String, String> dbParameters = db.getParameters();
+    if ((dbParameters != null) && (dbParameters.containsKey(ReplConst.REPL_FAILOVER_ENABLED))) {
+      return ReplConst.TRUE.equals(dbParameters.get(ReplConst.REPL_FAILOVER_ENABLED));
+    }
+    return false;
```

Comment:
Does this single line suffice?
return dbParameters != null && ReplConst.TRUE.equals(dbParameters.get(ReplConst.REPL_FAILOVER_ENABLED));

Reply:
also, ReplConst.TRUE.equals : do we need to handle case sensitiveness?